.github/workflows/node-release.yml

```diff
@@ -249,6 +249,9 @@ jobs:
           submodules: recursive
           fetch-depth: 0
 
+      - name: Get Latest Version
+        run: git pull
+
```

Comment:
Do you need two separate steps at all?

Reply:
Sorry mean two separat jobs...

Reply:
ya, the versioning has to be run in a separate job before everything. the build are basically running 3 seperate builds for each OS in the matix, so they each need the same version

Reply:
you can't really put the version bump in the OS matrix part of the workflow, otherwise it would run once per OS, bumping the version multiple times.